spectrum/checks.py

```diff
@@ -839,6 +839,12 @@ def _build_url(path, host):
     'articles/{id}/elife-{id}-v{version}.pdf',
     'articles/{id}/elife-{id}-v{version}.pdf'
 )
+PACKAGING_BUCKET = BucketFileCheck(
+    aws.S3,
+    SETTINGS['bucket_packaging'],
+    '{vendor}/{folder}/elife{id}.xml',
```

Comment:
A note here, for Crossref I changed this so the files added to the outbox include the version number, so there the files are in the format e.g. ``elife-29739-v2.xml``. I may do this for the Pubmed outbox files too in the ``ScheduleDownstream`` activity in the future, where this strict naming match will not delete the right files. Currently though this is correct, I just wanted to mention this may change soon.

Reply:
You know where to change it with a pull request when that happens :)